### ISSUE TITLE
Anti-human ferritin peptide antibody

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10344]
**Preferred term label:**

**Synonyms**
antibodies against the human ferritin peptide

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies (immunoglobulins) in the blood circulation that react against human ferritin peptide

PMID: 36344353 It has also been shown that antibodies against the human ferritin peptide are present in more than 90% of patients with active GCA, making it a potentially useful diagnostic marker. 

PMID: 22228484 Antibodies against the ferritin peptide were present in up to 92% of untreated, active GCA and PMR patients. They can be useful as a diagnostic marker of PMR and GCA (Giant cell arteritis)

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Giant cell arteritis ORPHA:397

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000508
- label: Anti-human ferritin peptide antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:22228484|PMID:36344353
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against the human ferritin peptide.
- comment: It has also been shown that antibodies against the human ferritin peptide are present in more than 90% of patients with active giant cell arteritis, making it a potentially useful diagnostic marker.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10344
- synonyms:
